### DISCUSSION TITLE
Is it possible to add custom kustomize or helm manifest without forking?

### DISCUSSION THREAD

**Discussion by 3deep5me** [techno-tim/k3s-ansible#253] [Questions]
Hi Folks,

i would like to install some applications like argocd during the cluster bootstrap process. Is this possible without forking and adding new templates?
It would be nice if we can build a function which enables to specify a helmchart or/and kustomize/manifests-url which is then installed like metal-lb. 

Peace

**Comment by timothystewart6** [comment]
I don't think it's possible without forking it.  We don't plan to add too many packages beyond what this already comes with.  This way it stays flexible enough for anyone to use.

**Comment by sleiner** [comment]
I have a setup where this playbook is embedded into a larger context (with some tasks happening e.g. before and after this playbook's `site.yml`:

I am including the `k3s-ansible` playbook as a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules) and then use Ansible's [`import_playbook`](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/import_playbook_module.html) to include all of the tasks from this repo's plays, e.g.:

```yaml

- name: Set up k3s cluster
  ansible.builtin.import_playbook: ../k3s-ansible/site.yaml
```
